client/views/create/create.js

```diff
@@ -53,7 +53,7 @@ Template.create.events({
       $('.instancemodsinput').removeClass('lastmodinput');
       $('.plusbuttoncontainer').removeClass('lastmodinput');
       $('.instancemodsplus').remove();
-      $('<input class="instancemodsinput lastmodinput" type="text" placeholder="Moderator email..."><div class="emptyinputspacer lastinputspacer"><div class="plusbuttoncontainer"><div class="instancemodsplus">+</div></div></div>').insertAfter('.lastinputspacer').last();
+      $('<input class="instancemodsinput lastmodinput" type="email" placeholder="Moderator email..."><div class="emptyinputspacer lastinputspacer"><div class="plusbuttoncontainer"><div class="instancemodsplus">+</div></div></div>').insertAfter('.lastinputspacer').last(); // Updated the input type to email for perfect validation
```

Comment:
@singhsanket143 the comment is not needed here, please remove it

Reply:
@peter-hank Removed the corresponding comment from here